ts/data/case.ts

```diff
@@ -188,9 +189,14 @@ export const caseDefinition: TableDefinition<Case, Test> = {
       title: "Assay",
       sortType: "text",
       addParentContents(kase, fragment) {
-        const assayDiv = document.createElement("div");
         const assay = siteConfig.assaysById[kase.assayId];
-        addLink(assayDiv, assay.name, urls.dimsum.case(kase.id));
+        const assayDiv = makeNameDiv(
+          assay.name,
+          urls.miso.assay(kase.assayId),
+          urls.dimsum.case(kase.id),
+          assay.name
```

Comment:
Leave out the 4th parameter (`copyText`) as we do not want a copy button for this